exonum/src/proto/schema/exonum/consensus.proto

```diff
@@ -29,11 +29,11 @@ import "google/protobuf/timestamp.proto";
 // Keep in mind, that:
 // hash(SignedMessage) = hash(payload || key.bytes || sign.bytes)
 message SignedMessage {
-  // Message payload.
+  // Payload of the message.
   bytes payload = 1;
-  // Message author.
+  // Public key of the author of this message.
   exonum.crypto.PublicKey author = 2;
-  // Digital signature.
+  // Digital signature of the payload which creates with a secret key of the author of this message.
```

Comment:
```suggestion
  // Digital signature of the `payload` created by a `SecretKey` of the author of the message.
```

Reply:
```suggestion
  // Digital signature over `payload` created with `SecretKey` of the author of the message.
```

Reply:
I'm not sure if we really need a ` here. As I know the purpose of this docs is for a client code. @dmitry-timofeev what do you think ?

Reply:
alright, I do not mind to eliminate "`". In fact, I would even prefer that variant. In that case the comment should look like this:
> // Digital signature over the payload created with a secret key of the author of the message.